src/Parser/BrewmasterMonk/Modules/Spells/IronSkinBrew.js

```diff
@@ -17,9 +17,6 @@ class IronSkinBrew extends Module {
   hitsWithoutIronSkinBrew = 0;
   damageWithoutIronSkinBrew = 0;
 
-  // This value is just for dev purposes to check the total, its not shown on the tooltip.
```

Comment:
If it's for dev purposes you can add a dev statement, but please don't test it, it makes it feels like it's public API that is required, making it hard for someone else / future you from coming back and refactoring this as they won't understand the purpose of this easily